synthtool/README.md

```diff
@@ -13,26 +13,24 @@ This tool helps to generate and layout cloud client libraries. Synthtool runs th
 
 3. **Docker**  Some synth.py files require [Docker] to generate code.
 
+4. Add this repo to your **PYTHONPATH**.
+
+    ```
+    cd synthtool
+    export PYTHONPATH=`pwd`
```

Comment:
Is this really how we suggest people install it locally? Personally, I `python3 -m pip install -e .`

Reply:
That's good point. Unless we recommend avoiding a global package install? Could use `venv` of course.

Reply:
Done.